src/NuGet.Core/NuGet.Protocol.Core.Types/NuGetHttpHeaderUtility.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Net.Http;
+using NuGet.Common;
+
+namespace NuGet.Protocol.Core.Types
+{
+    public class NuGetHttpHeaderUtility
+    {
+        public const string ApiKeyHeader = "X-NuGet-ApiKey";
```

Comment:
Found one more occurrence of the same constant. We should consider having one `ProtocolConstants` class to share common strings.